resources/views/inputs/tile-selection-option.blade.php

```diff
@@ -40,11 +43,17 @@ class="form-checkbox tile-selection-checkbox"
         />
     @endif
 
-    <div class="flex flex-col justify-center items-center space-y-2 h-full font-semibold">
+    <div class="{{ $iconWrapper }} font-semibold">
```

Comment:
You can move `font-semibold` to be the `$titleClass` default as it only applies to this